### ISSUE TITLE
Add array of supported XEPs

### ISSUE THREAD

**Issue by chrisballinger** [ChatSecure/xmpp-server-list#1]
Use XEPper tool to get server features: https://davidchiles.github.io/XEPper/

@davidchiles Can we modify XEPper to also produce JSON output of supported XEPs? 

This will be useful in the future for determining if a server has XEP-0357 / push support.

``` json
{
"domain": "dukgo.com",
"extensions": ["XEP-0198", "XEP-0077", "RFC-6121"]
}
```

**Comment by chrisballinger** [comment]
We now have a much better way of doing this client-side

**Comment by dllud** [comment]
Nowadays it would be better to use the [XMPP Compliance Tester](https://github.com/iNPUTmice/ComplianceTester) by @iNPUTmice as it scans many more XEPs. You wouldn't even need to run it, you could just parse the available [reports](https://github.com/iNPUTmice/ComplianceTester/tree/master/reports).

**Comment by chrisballinger** [comment]
Agreed! We should write a script that pulls in the results from the reports because running the compliance tester is a pain. Right now the only critical XEPs are 0357 and 0363, but that will probably change over time.
